### PR TITLE
Update chapter 1 and add draw.io diagram

### PR DESCRIPTION
**Pull request by Krugo-D**

### CODE REVIEW

**Comment by aybehrouz** [comment]
Thanks. But you are actually updating Abstract not Chapter 1. 🙂

File: tex/A.tex [hunk 1]
```diff
@@ -52,24 +52,19 @@
...
+
+        The Argennon cloud consists of a Committee of Delegates (CoD) and Publicly Verifiable Cloud (PVC) servers. 
+        The CoD is elected per term by Proof of Stake and is responsible for processing transactions and creating blocks.
+        After validator assemblies have verified the validity of these blocks they are stored in the PVC servers.
+        PVC servers are publicly verifiable. Computational Integrity (CI) is
+        achieved by using succinct cryptographic proofs (STARK/SNARK) and data integrity is guaranteed by cryptographic accumulators. 
+        At the same time, a smart clustering algorithm keeps the bandwidth usage and the overhead of cryptography manageable
         for validators.
 
-        The Argennon protocol strongly incentivizes the formation of a permission-less network of Publicly Verifiable
```
Comment (aybehrouz) [inline tex/A.tex:66]: You've totally removed the important fact that the Argennon cloud is permission-less. Any one can create a server and help the Argennon cloud. This makes the cloud decentralized.

File: tex/A.tex [hunk 2]
```diff
@@ -52,24 +52,19 @@
...
-        Cloud (PVC) servers. A PVC server in Argennon, is a conventional data server which uses its computational and
-        storage resources to help the Argennon network process transactions. This encourages the development
-        of conventional networking, storage and compute hardware, which can benefit all areas of information technology.
-        This contrasts with the approach of some older blockchains that incentivized the development of a totally
-        useless technology of hash calculation.
```
Comment (aybehrouz) [inline tex/A.tex:71]: agree, we should remove this. (line 70-71)

**Review by aybehrouz — CHANGES_REQUESTED** [review]

File: tex/A.tex [hunk 3]
```diff
@@ -52,24 +52,19 @@
...
-        democratic blockchain and one of the most secure existing platforms.
-
-        The Argennon cloud is trustless and publicly verifiable. Computational Integrity (CI) is
-        achieved by using succinct cryptographic proofs (STARK/SNARK)
-        and data integrity is guaranteed by cryptographic accumulators. At the same time, a smart
-        clustering algorithm keeps the bandwidth usage and the overhead of cryptography manageable
+        outsourced to the Argennon cloud while normal personal computers or smartphones, with limited hardware
+        capabilities, can validate transactions and actively participate in the consensus protocol.
+
+        The Argennon cloud consists of a Committee of Delegates (CoD) and Publicly Verifiable Cloud (PVC) servers.
```
[the remark this answers is not in the document]
Comment (aybehrouz) [reply tex/A.tex:58]: or I don't know maybe CoD doesn't sound that bad... 😄

[the remark this answers is not in the document]
Comment (Krugo-D) [reply tex/A.tex:58]: Ah thanks for clearing that up. I do think the CoD is a fine abbreviation of an important aspect of ARG. Some people might argue that it is a centralized point of failure (wrt uptime of the blockchain).

PS: I quit playing after Modern Warfare :p

File: tex/A.tex [hunk 4]
```diff
@@ -52,24 +52,19 @@
...
-        outsourced to the cloud and normal personal computers or smartphones, with limited hardware
-        capabilities, can validate transactions and actively
-        participate in the Argennon consensus protocol. This property makes Argennon a truly decentralized and
-        democratic blockchain and one of the most secure existing platforms.
-
-        The Argennon cloud is trustless and publicly verifiable. Computational Integrity (CI) is
-        achieved by using succinct cryptographic proofs (STARK/SNARK)
-        and data integrity is guaranteed by cryptographic accumulators. At the same time, a smart
-        clustering algorithm keeps the bandwidth usage and the overhead of cryptography manageable
+        outsourced to the Argennon cloud while normal personal computers or smartphones, with limited hardware
```
[the remark this answers is not in the document]
Comment (Krugo-D) [reply tex/A.tex:55]: Agreed

File: tex/A.tex (hunk 1 above)
Comment (Krugo-D) [reply tex/A.tex:66]: I thought the incentives were sufficiently explained in detail later on. But yeah we could briefly state it in the introduction already

File: tex/A.tex (hunk 3 above)
[the remark this answers is not in the document]
Comment (aybehrouz) [reply tex/A.tex:58]: Yeah, We've had that discussion before, you may find it interesting: https://github.com/aybehrouz/argennon/discussions/9

File: tex/A.tex (hunk 1 above)
Comment (aybehrouz) [reply tex/A.tex:66]: you mean abstract 😁

File: tex/sections/intro.tex [hunk 5]
```diff
@@ -19,7 +19,7 @@
 
 It appears that the only solution to this important vulnerability is to effectively participate all the stake of the
 system in the consensus protocol. This large participation makes the protocol resilient against bribery or
-collusion because the adversary would need to spend unrealistic amount of money to bribe enough users.
+collusion because the adversary would need to spend unrealistic amounts of money to bribe enough users.
```
Comment (aybehrouz) [inline tex/sections/intro.tex:22]: Thanks!

**Comment by aybehrouz** [comment]
There is nothing wrong with opening several pull requests. Please let this pull request be focused on the abstract.

**Pull request merged**